src/Badge/index.js

```diff
@@ -0,0 +1,81 @@
+// @flow
+import * as React from "react";
+import styled from "styled-components";
+
+import defaultTokens from "../defaultTokens";
+import TYPE_OPTIONS from "./consts";
+
+import type { Props } from "./index";
+
+const StyledBadge = styled(({ className, children }) => (
+  <div className={className}>{children}</div>
+))`
+  font-family: ${({ theme }) => theme.orbit.fontFamily};
+  display: inline-flex;
+  box-sizing: border-box;
+  justify-content: center;
+  align-items: center;
+  height: 24px; // TODO: create token
+  line-height: 24px; // TODO: create token
+  width: ${({ circled }) => circled && "24px"}; // TODO: create token
+  font-size: ${({ theme }) => theme.orbit.fontSizeTextSmall};
+  font-weight: ${({ theme }) => theme.orbit.fontWeightMedium};
+  background-color: ${({ tokens, type }) => tokens.background[type]};
+  color: ${({ tokens, type }) => tokens.color[type]};
+  border-radius: 12px; // TODO: create token borderRadiusBadge
```

Comment:
In documentation is: 

*circled*: If true, the Badge will have rounded corners.

but I think the corners are rounded always

Reply:
Fixed docs.